### DISCUSSION TITLE
Remove slice home page

### DISCUSSION THREAD

**Discussion by stecog** [cantaloupe-project/cantaloupe#493] [Q&A]
Hi, is there any way to delete the Cantaloupe home page? In a production environment I would like if you access the cantaloupe URL you would not see the slice and the version, is that possible?

Thanks

**Comment by adolski** [comment]
Hi, there is no configuration for this--you would have to block it using a reverse proxy.

I'm open to adding a configuration switch to disable this page if there is demand for it.

(Sort of related to #220)
